src/network/core_utils.rs

```diff
@@ -715,7 +715,18 @@ impl CoreUtils {
                 ))
             }
         }
-
+        // TODO, this should only be done for IPv6
```

Comment:
Why is this inside `configure_bridge`.

Reply:
I think we should have a different function for this and call `sysctl` in `core.rs`

Reply:
Actually I'd also like to request why is this not possible in `setup.rs` or with firewall where other `sysctls` are made if `driver` is `bridge`.

Reply:
Yes the systctl are part of the bridge setup, I am not sure if need to set them for other drivers. Also it is important the the systctl is set before the interface is set up/get the ip assigned.